client/components/domains/register-domain-step/index.jsx

```diff
@@ -1167,12 +1177,10 @@ class RegisterDomainStep extends React.Component {
 		const suggestions = this.getSuggestionsFromProps();
 
 		// the search returned no results
-		if (
-			suggestions.length === 0 &&
-			! this.state.loadingResults &&
-			this.props.showExampleSuggestions
-		) {
-			return this.renderExampleSuggestions();
+		if ( suggestions.length === 0 && ! this.state.loadingResults ) {
+			if ( this.props.showExampleSuggestions ) {
```

Comment:
Does this `if` statement change do anything different to the previous version?

Reply:
This just restores to the previous version and does nothing different. Reason for restoring this is that we decided to retain the domain example box.